README.md

```diff
@@ -1,42 +1,165 @@
 # HaaS
 HPCC as a service: HPCC cluster management in clouds.
 
+## Requirements
```

Comment:
This is required for development. But is all this required if only using `setup.py`?